src/main/java/core/basesyntax/validators/ReportValidator.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.validators;
```

Comment:
package name should be in singular not plural
```suggestion
package core.basesyntax.validator;
```